sitemedia/js/main.js

```diff
@@ -7,7 +7,7 @@ const application = Application.start();
 const context = require.context(
     "./controllers",
     true,
-    /^.*\/(?!iiif_controller)(?!annotation_controller).*\.js$/
+    /^.*\/(?!iiif_controller)(?!annotation_controller)(?!alert_controller)(?!transcription_controller).*\.js$/
```

Comment:
would you add a comment here about what this is doing? this is what you mean about the double error?

Reply:
Yes, the double error was that the stimulus controller for alerts was being loaded twice (once in the `main` bundle and once in the `annotation` bundle).